lightning/src/util/events.rs

```diff
@@ -923,13 +937,15 @@ impl MaybeReadable for Event {
 					let mut payment_preimage = None;
 					let mut payment_secret = None;
 					let mut amount_msat = 0;
+					let mut receiver_node_id = None;
 					let mut _user_payment_id = None::<u64>; // For compatibility with 0.0.103 and earlier
 					read_tlv_fields!(reader, {
 						(0, payment_hash, required),
 						(2, payment_secret, option),
 						(4, amount_msat, required),
 						(6, _user_payment_id, option),
 						(8, payment_preimage, option),
+						(11, receiver_node_id, option),
```

Comment:
This probably should be the next odd type, i.e., 1? (Although, depends if this will be merged before or after #1856, as this also adds some new fields here.